onchain/rollups/contracts/portals/ERC1155BatchPortal.sol

```diff
@@ -14,11 +15,19 @@ import {InputEncoding} from "../common/InputEncoding.sol";
 ///
 /// @notice This contract allows anyone to perform batch transfers of
 /// ERC-1155 tokens to a DApp while informing the off-chain machine.
-contract ERC1155BatchPortal is InputRelay, IERC1155BatchPortal {
+contract ERC1155BatchPortal is IERC1155BatchPortal, InputRelay {
```

Comment:
I change the inheritance order here because the reference order of the `super` keyword is from right to left